3-Authorization-II/2-call-api-b2c/README.md

```diff
@@ -76,112 +101,115 @@ For more information and potential issues, see: [HTTPS in .NET Core](https://doc
-   - Select the **Add permissions** button at the bottom.
-   - Finally, click on the **Grant admin consent** button at the top.
-
-#### Configure the client app (msal-angular-spa) to use your app registration
+    1. In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-angular-spa`.
+    1. Under **Supported account types**, select **Accounts in any identity provider or organizational directory (for authenticating users with user flows)**
+    1. Select **Register** to create the application.
+1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select the **Authentication** blade to the left.
+1. If you don't have a platform added, select **Add a platform** and select the **Single-page application** option.
+    1. In the **Redirect URI** section enter the following redirect URIs:
+        1. `http://localhost:4200`
+        1. `http://localhost:4200/auth`
+    1. Click **Save** to save your changes.
+1. Since this app signs-in users, we will now proceed to select **delegated permissions**, which is is required by apps signing-in users.
+    1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs:
+    1. Select the **Add a permission** button and then:
+    1. Ensure that the **My APIs** tab is selected.
+    1. In the list of APIs, select the API `msal-dotnet-api`.
+        1. Select **delegated permissions**, which is requested by apps that signs-in users.
+        1. In the **Delegated permissions** section, select **ToDoList.Read** and **ToDoList.ReadWrite** in the list. Use the search box if necessary.
+    1. Select the **Add permissions** button at the bottom.
+
+##### Configure the client app (msal-angular-spa) to use your app registration
 
 Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
 
 > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
 
 1. Open the `SPA\src\app\auth-config.ts` file.
-1. Find the key `clientId` and replace the existing value with the application ID (clientId) of `msal-angular-spa` app copied from the Azure portal.
-1. Find the key `protectedResources.todoListApi.scopes` and replace the existing value with the scope of the web API that you have just exposed during the web API registration steps, for example `https://{tenantName}.onmicrosoft.com/{service_clientId}/demo.read`
-
+1. Find the key `Enter_the_Application_Id_Here` and replace the existing value with the application ID (clientId) of `msal-angular-spa` app copied from the Azure portal.
+1. Find the key `Enter_the_Tenant_Info_Here` and replace the existing value with your Azure AD tenant domain, ex. `contoso.onmicrosoft.com`.
```

Comment:
These aren't in the file.

Instead of `Enter_the_...` there's a GUID.

Reply:
And a reference to the `b2cPolicies` section.

Reply:
Yeah we need to tweak the codegen a little to use a different text in the case of B2C -will update

Reply:
done!